.github/ISSUE_TEMPLATE/bug-report.yaml

```diff
@@ -0,0 +1,81 @@
+name: Bug report
+description: Report incorrect behavior in the Modin library
+title: 'BUG: '
+labels: ['bug 🦗', 'Triage 🩹']
+
+body:
+  - type: checkboxes
+    id: checks
+    attributes:
+      label: Modin version checks
+      options:
+        - label: >
+            I have checked that this issue has not already been reported.
+          required: true
+        - label: >
+            I have confirmed this bug exists on the latest version of Modin.
+          required: true
+        - label: >
+            I have confirmed this bug exists on the main branch of Modin.
```

Comment:
We should tell people how to do this. Maybe just link here: https://modin.readthedocs.io/en/stable/getting_started/installation.html#installing-from-the-github-master-branch

I think a lot of people will just skip this step anyway, but having the link makes it more likely that they'll try.

also, should we mark this step as required, too?

Reply:
Yeah perhaps we don't have to require the latest master since most folks will be doing things via `pip install`. It's not like we're getting a ton of these issues either way.

Reply:
@mvashishtha I think we have a decent number of people who can't test on latest master, so I think it's ok to not have it there for now. If we start getting lots of bug requests that were solved in the latest version then we can do something about it.